librz/util/sdb/src/array.c

```diff
@@ -60,22 +60,26 @@ static inline int int_cmp(const void *a, const void *b) {
 	return 0;
 }
 
-RZ_API ut64 sdb_array_get_num(Sdb *s, const char *key, int idx) {
+RZ_API bool sdb_array_get_num(Sdb *s, const char *key, int idx, ut64 *out_number) {
```

Comment:
```suggestion
RZ_API bool sdb_array_get_num(RZ_NONNULL RZ_BORROW Sdb *s, RZ_NONNULL const char *key, int idx, RZ_OUT RZ_NONNULL ut64 *out_number) {
```

Also add the `rz_return_val_if_fail()` assert for the pointer arguments. And document it.